.github/workflows/ci.yml

```diff
@@ -31,13 +31,6 @@ jobs:
         with:
           name: action.yml
           path: action.yml
-      - id: vars
-        run: |
-          if [[ "${{ github.event_name }}" == "pull_request" ]]; then \
```

Comment:
Great if this can be avoided in all workflows where you’d want to support both branch pushes and PRs.

Reply:
Yeah. It was a bit of an oversight that I didn't add this into the action earlier. Thanks for raising the issue!